src/lib/bootstrap.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
you mentioned you have a similar file in santa that contains the version number. I was wondering if we should include something like that as well? Might help with future debugging in prod?

Reply:
I've added that now. Take a look—I use Rollup to generate a virtual config file we can import.